tests/test_startup.py

```diff
@@ -245,26 +245,39 @@ def _mock_folder_listing(self):
         else:
             yield
 
-    def _get_plugin_environment(self, dcc_path):
+    def _join_existing_environment(self, dcc_path, startup_path):
+        """
+        Returns the pre-existing environment paths for dcc_path, os.pathsep joined with the startup_paths
```

Comment:
Yeah I'm not sure. I put startup path as all the functions that would call this would pass a single string path like `"/my_path/"`

I then put it in a list in the method. I wasn't sure if it should expect a list of strings  or a pathseparated string (in which case it would be startup_paths), or just a single path in a string.
I've based it upon the latter, so I went with startup_path.